src/app/dal/https-client.js

```diff
@@ -35,9 +40,9 @@ const httpsRequest = (
         });
         incomingMessage.on('end', () => {
           try {
-            if (validResponsePayload.statusCode === 204) {
-              resolve('');
```

Comment:
Don't resolve if status code is 204 since that is like a return. Presumably this was here so that we didn't JSON.parse an empty payload...